torch/_inductor/codegen/cpp_micro_gemm.py

```diff
@@ -609,13 +609,9 @@ class CppMicroGemmAMX(CppMicroGemm):
     int b_tile_ptr_stride = ldb * {{vnni_size}};
 
     auto load_B_row = [&]({{input2_t}}* src, {{input_t}}* dst) {
-        {{kernel.unroll_pragma(2)}}
-        for (int i = 0; i < 2; i++) {
-            // int8 -> int32 -> fp32 -> bf16
-            auto b32 = at::vec::convert_to_int32<int8_t>(src + i * 16);
-            auto b_bf16 = at::vec::convert<{{input_t}}>(b32);
-            b_bf16.store(dst + i * 16);
-         }
+        auto b_int8 = at::vec::Vectorized<int8_t>::loadu(src, static_cast<int64_t>(32));
+        auto b_bf16 = at::vec::convert<{{input_t}}>(b_int8);
+        b_bf16.store(dst);
```

Comment:
Thank you! I didn't know this was possible with `at::vec`, since the conversion happens from `int8 -> int32 -> fp32 -> bf16`.

Reply:
@leslie-fang-intel, do you know if such a change is possible for the AVX512 micro-kernel as well, so that it could load multiple vector registers of `B` at a time? Thanks!

Currently, it loads only one vector register of B at a time

<img width="475" alt="image" src="https://github.com/user-attachments/assets/2f6b4982-f177-4a6a-91f3-5e202bdad1d8">

Reply:
yes, I guess so. Probably something like
```
        auto b_int8 = at::vec::Vectorized<int8_t>::loadu(src, static_cast<int64_t>(16));  // load first 128 bits of 16 X int8
        auto b_fp32 = convert_int8_to_float<int8_t>(b_int8); // CVT to 16 X FP32
```
Could you take a try if it benefits the AVX Micro GEMM performance?

Reply:
But this one is similar to the current implementation.